src/arrayMethodJoin.js

```diff
@@ -4,8 +4,22 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let answerStrig = '';
+
+    for (let i = 0; i < this.length; i++) {
+      if (this[i] !== undefined && this[i] !== null) {
+        answerStrig += this[i];
+      } else {
+        answerStrig += '';
+      }
```

Comment:
Good place for the ternary operator. Or even better - the nullish coalescing operator.

Reply:
i really don't understand how to use this operator on my issue. can u please explain it me?